wagtail/core/models.py

```diff
@@ -1833,6 +1833,30 @@ def can_publish_pages(self):
         """Return True if the user has permission to publish any pages"""
         return self.publishable_pages().exists()
 
+    def unlockable_pages(self):
+        """Return a queryset of the pages that this user has permission to unlock"""
+        # Deal with the trivial cases first...
+        if not self.user.is_active:
+            return Page.objects.none()
+        if self.user.is_superuser:
+            return Page.objects.all()
```

Comment:
If we're keeping this (which we may not be, given the other point...) we should make it consistent about whether it's returning pages that are currently locked (in which case we want a `filter(locked=True)` here and on the 'unlock any page' case) or ones that could hypothetically be unlocked in the case that they were locked :-)

Reply:
This does the latter at the moment - it only includes pages a user has personally locked because the logic for unlocking allows them to unlock them if they are the locking user, even if they wouldn't normally have the unlock permission.